src/main/resources/db/changelog/db.changelog-RDM-8463.xsl

```diff
@@ -0,0 +1,17 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <changeSet id="rdm-8463" author="suman.sree@hmcts.net">
```

Comment:
you need to add the changeset to : 
https://github.com/hmcts/ccd-data-store-api/blob/fc07f8b55fb67379b2e7db457fa08d347d004c7f/src/main/resources/db/changelog/db.changelog-master.xml#L1
for it to be executed